src/Sylius/Component/Addressing/Model/Country.php

```diff
@@ -131,7 +131,6 @@ public function removeProvince(ProvinceInterface $province)
     {
         if ($this->hasProvince($province)) {
             $this->provinces->removeElement($province);
-            $province->setCountry(null);
```

Comment:
Cause province __can't__ exist without country, and this must be covered by Doctrine relation, otherwise database would be polluted with non-matching provices.